src/model/transaction/getSampleStateForTesting.js

```diff
@@ -13,19 +13,19 @@
 
 'use strict';
 
-var BlockMapBuilder = require('BlockMapBuilder');
-var CharacterMetadata = require('CharacterMetadata');
-var ContentBlock = require('ContentBlock');
-var ContentState = require('ContentState');
-var EditorState = require('EditorState');
-var Immutable = require('immutable');
-var SampleDraftInlineStyle = require('SampleDraftInlineStyle');
-var SelectionState = require('SelectionState');
+const BlockMapBuilder = require('BlockMapBuilder');
+const CharacterMetadata = require('CharacterMetadata');
+const ContentBlock = require('ContentBlock');
+const ContentState = require('ContentState');
+const EditorState = require('EditorState');
+const Immutable = require('immutable');
+const SampleDraftInlineStyle = require('SampleDraftInlineStyle');
+const SelectionState = require('SelectionState');
 
-var {BOLD, ITALIC} = SampleDraftInlineStyle;
-var ENTITY_KEY = '123';
+const {BOLD, ITALIC} = SampleDraftInlineStyle;
+const ENTITY_KEY = '1';
```

Comment:
Just  curious - why change this to 1?

Reply:
TLDR:

Entitymap is indexed based on insertions (1,2,3) when processed on the convertStateToRaw, having the previous `123` as key meant we would not retrieve the key correctly from the entitymap

Reply:
for reference:

https://github.com/facebook/draft-js/blob/master/src/model/encoding/convertFromDraftStateToRaw.js#L27

```
function convertFromDraftStateToRaw(
  contentState: ContentState,
): RawDraftContentState {
  var entityStorageKey = 0;
```

and https://github.com/facebook/draft-js/blob/master/src/model/encoding/convertFromDraftStateToRaw.js#L56


```
  // Flip storage map so that our storage keys map to global
  // DraftEntity keys.
  var entityKeys = Object.keys(entityStorageMap);
  var flippedStorageMap = {};
  entityKeys.forEach((key, jj) => {
    var entity = contentState.getEntity(DraftStringKey.unstringify(key));
    flippedStorageMap[jj] = {
      type: entity.getType(),
```